packages/client/src/containers/DropdownGroup/DropdownGroup.Container.js

```diff
@@ -2,11 +2,34 @@ import React from 'react';
 import ViewDropdown from '../../components/ViewDropDown/ViewDropdown.component';
 import './DropdownGroup.Style.css';
 
-export const DropdownGroup = () => {
+export const DropdownGroup = (productId) => {
   const Colors = ['Black', 'White', 'Blue', 'Half-white'];
   const Sizes = ['Small', 'Medium', 'Large', 'XL', 'XXL'];
   const Quantities = [1, 2, 3, 4, 5, 6, 7, 8, 9];
 
+  const variants = [
+    {
+      id: 9,
+      product_id: 7,
+      color: 'merun',
+      size: '3XL',
+      pictureUrl: 'packages/client/src/assets/jeans&shoes.jpg',
+      stock: 18,
+    },
+  ];
+
+  variants.forEach((variant) => {
+    if (!Colors.includes(variant.color)) {
+      Colors.push(variant.color);
+    }
+    if (!Sizes.includes(variant.size)) {
+      Sizes.push(variant.size);
+    }
+    if (!Quantities.includes(variant.stock)) {
+      Quantities.push(variant.stock);
+    }
+  });
```

Comment:
This should all be controlled by React `useState` so that it doesn't happen on each render.
Once you connect the API that might make more sense and consider using `Set`.